.gitignore

```diff
@@ -1,2 +1,4 @@
 *.pyc
 *.swp
+robots/BALANC3R/ev3dev
+robots/EDUCATOR/ev3dev
```

Comment:
@gregcowell is the code touching these files and that is why we need them in the .gitignore?

Reply:
will go ahead and merge as this is super minor and we can easily chop later if it isn't needed

Reply:
Sorry, I should have explained that. In order to use the latest devel branch of ev3dev-lang-python, I use a symbolic link (ev3dev) in the same directory as the example script. Because I rsync the code from my laptop to the EV3, the laptop also has the symbolic link. There's probably a tidier way to do this.

Reply:
FYI, you can add local repository-specific excludes to `.git/info/exclude`.

Reply:
Much better. Thanks!